java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3538,4 +3538,36 @@ public void bitcount(BaseClient client) {
             assertTrue(executionException.getCause() instanceof RequestException);
         }
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void setbit(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+
+        assertEquals(0, client.setbit(key1, 0, 1).get());
+        assertEquals(1, client.setbit(key1, 0, 0).get());
+
+        // Exception thrown due to the negative offset and is out of range
+        ExecutionException executionException =
+                assertThrows(ExecutionException.class, () -> client.setbit(key1, -1, 1).get());
+        assertTrue(executionException.getCause() instanceof RequestException);
+
+        // Exception thrown due to the offset exceeding the offset limit of 2^32
+        executionException =
+                assertThrows(ExecutionException.class, () -> client.setbit(key1, 4294967296L, 1).get());
```

Comment:
I would consider removing this test.  It's unnecessary to test the limits of Redis, and this really just adds more overhead for our tests.  
Really what you're testing here is that 4294967296L is a valid Long value and can be sent through Protobuf... which isn't a very strong test.